src/components/votingListView/votingListView.js

```diff
@@ -144,54 +139,41 @@ class DelegateList extends React.Component {
     return message;
   }
 
-  getList(filteredList) {
-    return filteredList.map(item => (
-      <VotingRow key={item.address} data={item}
-        className={this.state.safariClass}
-        voteToggled={this.props.voteToggled}
-        voteStatus={this.props.votes[item.username]}
-      />
-    ));
-  }
-
   render() {
-    const showChangeSummery = this.state.showChangeSummery ?
-      styles.showChangeSummery : '';
     const filteredList = this.filter(this.props.delegates);
+    const { showChangeSummery, isDelegate, voteToggled, votes, t } = this.props;
     return (
       <Box className={`voting delegate-list-box ${showChangeSummery} ${styles.box}`}>
         <VoteUrlProcessor />
-        <Header
+        <VotingHeader
           setActiveFilter={this.setActiveFilter.bind(this)}
-          showChangeSummery={this.state.showChangeSummery}
-          isDelegate={this.props.isDelegate}
-          voteToggled={this.props.voteToggled}
+          showChangeSummery={showChangeSummery}
+          isDelegate={isDelegate}
+          voteToggled={voteToggled}
           search={ value => this.search(value) }
         />
         <section className={`${styles.delegatesList} delegate-list`}>
           <div className={styles.table}>
-            <ul className={`${styles.tableHead} ${grid.row}`}>
-              <li className={`${grid['col-md-1']} ${grid['col-xs-2']} ${styles.leftText}`}>{this.props.t('Vote', { context: 'verb' })}</li>
-              <li className={`${grid['col-md-1']} ${grid['col-xs-2']}`}>{this.props.t('Rank')}</li>
-              <li className={`${grid['col-md-3']} ${grid['col-xs-5']}`}>{this.props.t('Name')}</li>
-              <li className={`${grid['col-md-5']}`}>{this.props.t('Lisk ID')}</li>
-              <li className={`${grid['col-md-2']} ${grid['col-xs-3']} ${styles.productivity}`}>{this.props.t('Productivity')}</li>
-            </ul>
-            { this.getList(filteredList) }
+            <ListLabels t={t} styles={styles} grid={grid} status={showChangeSummery} />
```

Comment:
Why pass `styles={styles} grid={grid}` instead of importing them in `ListLabels`? Or going one step further, `ListLabels` could have its own css file. Not necessarilly in this PR.